Changes in the proposal template + meeting template (#32)
Babylonpartners/ios-playbook: TechnicalDocuments/Proposals/Template_Proposal_Meeting_Invitation.md

Hi all,

This meeting is to discuss the "<Proposal Title>" proposal authored by <Proposal's Author>.

1.	PR here <Link to the Proposal's PR>.
2.	Actual proposal here <Link to the Proposal's .md file>.

Meeting Etiquette:

1.	**Please fully read the proposal.**
  a.	If you haven’t read the proposal, or don’t care for the outcome, please don’t show up for the meeting.

> 2.	If you want to counter purpose, or raise a change, please provide those in written to all meetings participants beforehand.

Reviewer: ```suggestion
2.	If you want to make a counter proposal, or raise a change, please provide those in written to all meetings participants beforehand.
```

>   a.	Ideally this is done via a PR’s comment.

Reviewer: ```suggestion
  a.	Ideally this is done via a PR comment.
```

>   b.	Meetings notes (or PR's comments) can only be provided 2h before the meeting.

Reviewer: ```suggestion
  b.	Meetings notes (or PR comments) can only be provided 2h before the meeting.
```

3.	Questions should be asked on the PR, not during the meeting.
4.	**The purpose of the meeting is to act as a sync point and a final outcome for the proposal itself.**

>   a. Comments can be made, but at this point, the author should have a good idea if the proposal is to move forward or not.

Reviewer: ```suggestion
  a. Comments can be made, but at this point, the author should have a good idea if the proposal is going to move forward or not.
```
